specs/core/0_beacon-chain.md

```diff
@@ -388,7 +389,7 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
     'randao_commitment': 'hash32',
     # Initial custody commitment
     'custody_commitment': 'hash32',
-    # a BLS signature of this ``DepositInput``
+    # A BLS signature of this `DepositInput`
```

Comment:
Don't we generally do double back ticks in comments?

Reply:
The double backticks are only for docstring.